src/components/RecordTypesSelect/RecordTypesSelect.css

```diff
@@ -17,16 +33,95 @@
 }
 
 .item {
+  overflow: hidden;
   outline: none;
   cursor: pointer;
   border-radius: .5rem;
   padding: .5rem;
   background-color: var(--primary);
   color: #fff;
   font-weight: 700;
-  margin-bottom: .8rem;
 
   &:hover {
     background-color: color(var(--primary) whiteness(40%))
   }
 }
+
+.heading {
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+}
```

Comment:
Consider adding an alignment or paddings on the sides to components with this class, because headings "stick" to borders.